src/connection-manager/connections.ts

```diff
@@ -152,9 +151,10 @@ setInterval(() => {
 export default async function startConnections(): Promise<void> {
   if (!cmStarted) {
     cmStarted = true
+    await clearConnectionsDb();
```

Comment:
So, what happens when a websocket disconnects? Does disconnect still set `connected` in the DB to false?